presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcPageSink.java

```diff
@@ -197,6 +205,14 @@ public void abort()
         catch (SQLException e) {
             throw new PrestoException(JDBC_ERROR, e);
         }
+        finally {
+            try {
+                jdbcClient.rollbackCreateTable(handle);
```

Comment:
why is this needed if whole transaction is rollbacked, then table should be dropped as well, am I right?

Also `JdbcPageSink` has not created this table, it looks fishy that it drops this.

Reply:
The writes will be rolled back, but not the create table statement.

I think @findepi's suggestion makes sense here, as it accomplishes the same thing as this code but in the correct place.